setup.py

```diff
@@ -1,6 +1,8 @@
 from setuptools import setup, find_packages
 
 from byudml import __version__
+from byudml import __imputer_path__
+from byudml import __metafeature_path__
```

Comment:
Installation no longer works because the installation script now depends on the metafeature primitive, which depends on metalearn, which depends on being installed: a circular dependency. We need another solution to single source the primitive python_path.

Reply:
Source in byudml.__init__